src/tests/ftest/util/ServerUtils.py

```diff
@@ -33,18 +34,110 @@
 import signal
 import fcntl
 import errno
+import yaml
 
 from avocado.utils import genio
 
 sessions = {}
 
+DEFAULT_YAML_FILE = "src/tests/ftest/data/daos_server_baseline.yml"
+AVOCADO_YAML_FILE = "src/tests/ftest/data/daos_avocado_test.yml"
```

Comment:
This is the DAOS server configuration file, as modified to run the Functional tests in our CI clusters, yes?

Can we rename this variable appropriately then?  AVOCADO_YAML_FILE sounds to me like the Avocado test yaml file.  IOW, the `.yaml` file that goes along with a test `.py` file.

Also, the fact that this file or the one above it are yaml files I think are immaterial to what they are and you can probably just drop YAML from variable name.

Reply:
I will update the name